cmd/xl-storage-format-utils.go

```diff
@@ -34,6 +34,14 @@ func (v versionsSorter) Less(i, j int) bool {
 	return v[i].ModTime.After(v[j].ModTime)
 }
 
+type xlMetaV2VersionsSorter []xlMetaV2Version
+
+func (v xlMetaV2VersionsSorter) Len() int      { return len(v) }
+func (v xlMetaV2VersionsSorter) Swap(i, j int) { v[i], v[j] = v[j], v[i] }
+func (v xlMetaV2VersionsSorter) Less(i, j int) bool {
+	return getModTimeFromVersion(v[i]).After(getModTimeFromVersion(v[j]))
+}
```

Comment:
Use `sort.Slice`, faster than this and less code.

Reply:
> Use `sort.Slice`, faster than this and less code.

done